libs/agentc_core/agentc_core/util/ddl.py

```diff
@@ -107,12 +107,12 @@ def create_vector_index(
         )
 
     max_partition = (
-        os.getenv("AGENT_CATALOG_MAX_SOURCE_PARTITION")
+        int(os.getenv("AGENT_CATALOG_MAX_SOURCE_PARTITION"))
         if os.getenv("AGENT_CATALOG_MAX_SOURCE_PARTITION") is not None
-        else None
+        else 1024
     )
     index_partition = (
-        os.getenv("AGENT_CATALOG_INDEX_PARTITION")
+        int(os.getenv("AGENT_CATALOG_INDEX_PARTITION"))
```

Comment:
it is better to do os.getenv() once and store it in a variable rather than calling it twice